composer.json

```diff
@@ -9,13 +9,17 @@
         "psr",
         "psr-7"
     ],
+    "config": {
+        "sort-packages": true
+    },
     "extra": {
         "branch-alias": {
             "dev-master": "1.0-dev"
         }
     },
     "require": {
         "php": "^5.6 || ^7.0",
+        "symfony/console": "^3.2",
```

Comment:
Please also specify the current patch version used: we don't want people to start with 3.2.0, but rather 3.2.7 (current)